<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RetrackingDialog</class>
 <widget class="QDialog" name="RetrackingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Retrack Fix Blob</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="blobInfoGroup">
     <property name="title">
      <string>Fix Blob Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelBlobId">
        <property name="text">
         <string>Blob ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="blobIdValue">
        <property name="text">
         <string>-</string>
        </property>
        <property name="styleSheet">
         <string>font-weight: bold;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelBlobFrame">
        <property name="text">
         <string>Frame:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="blobFrameValue">
        <property name="text">
         <string>-</string>
        </property>
        <property name="styleSheet">
         <string>font-weight: bold;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelBlobPosition">
        <property name="text">
         <string>Position:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="blobPositionValue">
        <property name="text">
         <string>-</string>
        </property>
        <property name="styleSheet">
         <string>font-weight: bold;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="frameRangeGroup">
     <property name="title">
      <string>Retracking Frame Range</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelStartFrame">
        <property name="text">
         <string>Start Frame:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="startFrameSpin">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelEndFrame">
        <property name="text">
         <string>End Frame:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="endFrameSpin">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="autoRangeCheck">
        <property name="text">
         <string>Auto-determine range around Fix blob frame</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelRangeSize">
        <property name="text">
         <string>Range size (Â±frames):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="rangeSizeSpin">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="optionsGroup">
     <property name="title">
      <string>Retracking Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="replaceExistingCheck">
        <property name="text">
         <string>Replace existing track data in this range</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="extendTrackCheck">
        <property name="text">
         <string>Extend existing tracks if possible</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Ready to start retracking</string>
     </property>
     <property name="styleSheet">
      <string>color: #666; font-style: italic;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RetrackingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RetrackingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoRangeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>startFrameSpin</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoRangeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>endFrameSpin</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoRangeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>rangeSizeSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
